test/wasm2c/old-spec/bulk-memory-operations/imports.txt

```diff
@@ -0,0 +1,607 @@
+;;; TOOL: run-spec-wasm2c
```

Comment:
I think this is probably a Git issue (recreating the `old-spec/bulk-memory-operations/imports.txt` file along with the other three files in this directory). It might be best to rebase your commits on top of the main branch (rather than using a merge commit), which will make the GitHub "Files changed" view a lot simpler.